src/Composer/Command/ShowCommand.php

```diff
@@ -471,7 +472,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
                                 $replacement
                             );
                             $packageViewData['warning'] = $packageWarning;
+                            $packageIsAbandoned = true;
```

Comment:
Usually we export this as `false|true|"replacement/package"` so truthy values are abandoned, and false or unset is non-abandoned.

It just means it should here use this I guess:
```suggestion
                            $packageIsAbandoned = $latestPackage->getReplacementPackage() ?? true;
```

Reply:
So in case we want to stay compatible with how it is written to the `composer.json`, right?
Is this a strict requirement to reflect the `composer.json` information or can we add a dedicated column which provides a replacement suggestion instead?

Something like this is way more easy to parse IMHO:

```json
{
    "abandoned": bool,
    "replacementPackage": null|non-empty-string
}
```

But I will implement your suggestion if we want to keep in-sync with the `composer.json` `abandoned` flag ofc.

Reply:
Yeah it's mostly about consistency as it is represented like that in composer.json and in various packagist.org json APIs

Reply:
Got it, thanks for explanation. For consistency, I've changed the code appropriately.